manim/_config/main_utils.py

```diff
@@ -109,6 +110,9 @@ def parse_args(args: list) -> argparse.Namespace:
         return _parse_args_cfg_subcmd(args)
     elif subcmd == "plugins":
         return _parse_args_plugins(args)
+    elif args[1] == "--version":
+        print("Manim Community Edition v"+__version__)
```

Comment:
Why not as a f-string?

```suggestion
        print(f"Manim Community Edition v{ __version__ }")
```